jenkins.check_copyright.py

```diff
@@ -30,11 +30,11 @@
     exit(0)
 
 with open(file_name, 'r') as sourcefile:
-    for x in range(0,len(copyright_lines)):
+    for regex in regexes:
         # TODO add a check for empty files (dummy.js etc), as they cause the script to crash here
         line = next(sourcefile)
         line = line.rstrip()
-        matches = regexes[x].match(line)
+        matches = regexes[regex].match(line)
```

Comment:
Not quite right. We replaced the index-based for loop which used index `x` with a list iteration for loop which sets `regex` equal to each value of the list in sequence. `regexes[x]` should have been replaced with `regex`.

This line should be `matches = regex.match(line)`